### PR TITLE
Cleanup lmp-extending user guide

### PR DESCRIPTION
**Pull request by kprosise**
Typos fixed and style rules applied, other small changes to increase readability.

QA Steps: Checked rendered HTML, no issues seen. Ran linter within editor, no new issues discovered.

No related ticket, quick fix.

# PR Template and Checklist

Please complete as much as possible to speed up the reviewing process.

Readiness and adding reviewers as appropriate is required.

All PRs should be reviewed by a technical writer/documentation team and a peer.
If effecting customers—which is a majority of content changes—a member of Customer Success must also review.

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

Discovered small typo while working on something else, did additional cleanup at the same time.

## Checklist

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 41b27fa1a4e160f4da321f2b073b78e9732346da are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2671/docs/artifacts/html/index.html

File: source/user-guide/lmp-customization/linux-extending.rst [hunk 1]
```diff
@@ -155,16 +156,18 @@ The command ``bitbake-getvar`` can be used to see the value of some variables, a
   DISTRO_FEATURES="acl argp bluetooth ext2 ipv4 ipv6 largefile usbgadget usbhost wifi xattr zeroconf pci vfat modsign efi security tpm integrity seccomp pam usrmerge virtualization ptest
   alsa sota systemd"
 
-The log is generated using ``DISTRO="lmp"``. The ``DISTRO_FEATURES`` changed with seven operations and only one of them is for an override (``tegra``).
-The log also shows the file path and line for each operation.
+Using ``DISTRO="lmp"`` generates the log.
+``DISTRO_FEATURES`` changes seven operations and one of them is for an override (``tegra``).
```
Comment (angolini) [inline source/user-guide/lmp-customization/linux-extending.rst:160]: there are seven operations that change the DISTRO_FEATURES values. I'm not sure removing the passive voice here is equivalent. 

could you please review it?

**Review by angolini — APPROVED** [review]

File: source/user-guide/lmp-customization/linux-extending.rst (hunk 1 above)
Comment (kprosise) [inline source/user-guide/lmp-customization/linux-extending.rst:160]:
```suggestion
``DISTRO_FEATURES`` can change from seven operations, and one of them is for an override (``tegra``).
```

**Comment by doanac** [comment]
Docs for f66ab7ff77d3a1acb9615d14e6bdd52934532846 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2672/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 3e26cf93276edfbd6e4d5fb84d25893d58647b56 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2673/docs/artifacts/html/index.html

**Pull request merged**